setup.py

```diff
@@ -0,0 +1,38 @@
+import setuptools
+from setuptools import setup
+from distutils.util import convert_path
+import os
+
+main_ns = {}
+ver_path = convert_path('src/version.py')
+with open(ver_path) as ver_file:
+    exec(ver_file.read(), main_ns)
+VER = main_ns['__version__']
+
+requirementPath = os.path.dirname(os.path.realpath(__file__)) + '/requirements.txt'
+install_requires = []
+if os.path.isfile(requirementPath):
+    with open(requirementPath) as f:
+        install_requires = f.read().splitlines()
+
+setup(
```

Comment:
we should also include torch and it's minions in the installation, or at least validate their versions

Reply:
I think it might be a problem, since installing torch depends on the install cuda toolkit version. So guess we should either check this or demand torch & friends as a prerequisite

Reply:
Mmm I really don't know how to do this, can you look at the requirments of other torch dependant libraries? I tried to look here but honestly didn't understand what's going on
https://github.com/pytorch/captum/blob/master/setup.py

Reply:
As we discussed on slack, we can just a pytorch version as a requirement and the user will install it manually